### PR TITLE
Updates elements of Site object to add sample values where needed.

### PR DESCRIPTION
**Pull request by kevinkirkl3y**
I have reviewed the elements contained by the Site object and added sample values where necessary.

### CODE REVIEW

**Review by jrippingale — APPROVED** [review]
Yes! I love seeing this coming together!

File: Master-OB-OpenAPI.json [hunk 1]
```diff
@@ -5381,7 +5395,9 @@
             "description": "A product code is a standardized, unique human-readable identifier that is compact,and can be easily parsed. It consists of an entity code concatenated to a product specific identification string, separated by a hyphen. A product string is up to 16 characters, comprised of upper case letters, number, and hyphens (Ã¢â‚¬Ëœ-Ã¢â‚¬Ëœ).\nThe format of a product code is:\n[EntityCode]-[ProductString], e.g., UEE-E2000-3-XFM for company Universal Energy Enterprises with Entity Code UEE and Product String of E2000-3-XFM.    ",
             "x-ob-item-type": "StringItemType",
             "x-ob-usage-tips": "",
-            "x-ob-sample-value": {},
+            "x-ob-sample-value": {
+              "Value": "IQ7PLUS-72-ACM-US"
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:5399]: This isn't a valid product code example, per our description. The manufacturer is Enphase. "IQ7PLUS-72-ACM-US" is the Product String and needs to be prefixed by the Entity Code for Enphase.

This illustrates the concern I have about our Product code definition, that OB is requiring an Entity Code that a manufacturer doesn't use and that customers won't know.

File: Master-OB-OpenAPI.json [hunk 2]
```diff
@@ -6518,7 +6542,9 @@
             "x-ob-item-type": "DateItemType",
             "x-ob-item-type-group": "",
             "x-ob-usage-tips": "",
-            "x-ob-sample-value": {}
+            "x-ob-sample-value": {
+              "Value": "2022-12-28T19:49:26Z"
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:6546]: Splitting hairs here: the element is CommentDate, of DateItemType, the example is a full datetime value. In other instances, e.g., CertificationDate, the example is only a date string. Should examples make the distinction between date and datetime strings?

File: Master-OB-OpenAPI.json (hunk 1 above)
Comment (kevinkirkl3y) [reply Master-OB-OpenAPI.json:5399]: Update description to include entity code of 6 digits and no limit on product code.

Comment (kevinkirkl3y) [reply Master-OB-OpenAPI.json:5399]: Example: SMA-SB7.7-1SP-US-41-NWF

**Pull request merged**